pkg/build/controller/controller.go

```diff
@@ -220,7 +223,7 @@ func (bc *BuildPodController) CancelBuild(build *buildapi.Build, pod *kapi.Pod)
 		return err
 	}
 
-	glog.V(2).Infof("Build %s was successfully cancelled.", build.Name)
+	glog.V(2).Infof("Build %s/%s was successfully cancelled.", build.Namespace, build.Name)
```

Comment:
@smarterclayton if it is so, then I dont thing logs like:
- Updating build status ...
- Build 1234 has been generated from 4321 build configuration
- Error resolving image tag reference
- Failed to parse request URL
- Failed to extract information from webhook
- ....
should be in the Hail Mary group